### DISCUSSION TITLE
Ubuntu booting problem on Surface Laptop 4

### DISCUSSION THREAD

**Discussion by bazarovgit** [linux-surface/linux-surface#875] [Q&A]
Hello everyone,
I decided to install Ubuntu on my Surface Laptop 4. I'm aware that it's a bit more difficult to get everything to work as it should with this hardware, but I armed myself with an external keyboard and mouse and good will, but alas after installing the distro and trying to boot for the first time without booting from USB media I get this error screen and it won't go any further.

Any advice?

![47545E7E-8609-4265-BF54-C4F5BB3B6B87](https://user-images.githubusercontent.com/109483707/179423151-3315944c-5d8e-4afc-bfaa-3de42a8d3771.jpeg)

**Comment by NP-chaonay** [comment]
perhaps you have to doing the workaround before linux-surface is fully installed.
![image](https://user-images.githubusercontent.com/17851493/179446266-766f2555-9c36-4e4a-993d-675ca5ae8b2f.png)
